.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Feature request
+about: Suggest a new idea for Flutter
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- Thank you for using Flutter!
+
+     If you are looking for support, please check out our documentation
+     or consider asking a question on Stack Overflow:
+      * https://flutter.dev/
+      * https://api.flutter.dev/
+      * https://stackoverflow.com/questions/tagged/flutter?sort=frequent
+
+     If you have found a bug or if our documentation doesn't have an answer
+     to what you're looking for, then fill our the template below. Please read
+     our guide to filing a bug first: https://flutter.dev/docs/resources/bug-reports
+-->
+
+## Use case
+
+<!--
+     Please tell us the problem you are running into that led to you wanting
+     a new feature.
+
+     Is your feature request related to a problem? Please give a clear and
+     concise description of what the problem is.
+
+     Describe alternative solutions you've considered. Is there a package
+     on pub.dartlang.org that already solves this?
+-->
+
+## Proposal
+
+<!--
+     Succinctly describe what you would like Flutter to be able to do.
```

Comment:
nit: use a simpler word instead of Succinctly? (I had to look up what it means, I imagine many non-native english speakers will as well).

Reply:
Heh, that was the word GitHub recommended. :-)

I'll change it.